packages/thirdweb/package.json

```diff
@@ -214,6 +224,7 @@
     "@types/react": "18.2.17",
     "@vitejs/plugin-react": "^4.2.1",
     "@vitest/ui": "1.5.0",
+    "amazon-cognito-identity-js": "^6.3.12",
```

Comment:
how come only this one is in devDeps?

Reply:
removed. should i add all the peer deps in here? i think its working right now because of the legacy package

Reply:
you should add as peer + dev dependencies all dependencies that you import from in the code (and they should all be optional peer deps since they are only needed if react native is being used)